crates/cfxcore/core/src/consensus/mod.rs

```diff
@@ -1954,6 +1958,12 @@ impl ConsensusGraph {
                 Some(r) => r,
             };
 
+            // note: we only include gas limit for blocks that will pack eSpace
+            // tx(multiples of 5)
+            if b.block_header.height() % 5 == 0 {
+                total_gas_limit += b.block_header.gas_limit() * 5 / 10
```

Comment:
Put these magic number into a shared variable, since it may be changed in the future.
Shortcut * 5 / 10 as / 2 , and add some comment.